src/webgpu/util/texture/texel_data.ts

```diff
@@ -775,24 +775,32 @@ export const kTexelRepresentationInfo: {
             components.B ?? unreachable()
           ),
         ]).buffer,
-      // For the purpose of unpacking, expand into three "ufloat14" values.
       unpackBits: (data: Uint8Array) => {
-        return unpackRGB9E5UFloat((data[3] << 24) | (data[2] << 16) | (data[1] << 8) | data[0]);
```

Comment:
I think this was the only usage of this function, so maybe it should be deleted.

While checking that, I accidentally found that this is actually brand new in #2796. You should sync with @ben-clayton about it.

Reply:
I was using it for texture execution tests, which I'm still working on. Please don't remove it.